exasol_transformers_extension/udfs/sequence_classification_text_pair_udf.py

```diff
@@ -38,6 +36,16 @@ def run(self, ctx):
 
         self.clear_device_memory()
 
+    def _set_device(self, device_id: Optional[int]) -> None:
```

Comment:
same as above

Reply:
I thought this method will be the part of base class that we will handle in the ticket #19 
When refactoring the class hierarchy, we will not need to call UDF to test the base class